### PR TITLE
GEODE-8850: Revise security-client-auth-init property defn

### PR DESCRIPTION
**Pull request by karensmolermiller**

### CODE REVIEW

File: geode-docs/reference/topics/gemfire_properties.html.md.erb [hunk 1]
```diff
@@ -568,7 +568,7 @@ Any security-related (properties that begin with <code class="ph codeph">securit
 </tr>
 <tr>
 <td>security-client-auth-init</td>
-<td>Used for authentication. Static creation method returning an <code class="ph codeph">AuthInitialize</code> object, which obtains credentials for peers in a cluster. The obtained credentials should be acceptable to the <code class="ph codeph">Authenticator</code> specified through the security-peer-authenticator property on the peers.</td>
+<td>Used for authentication. Location of the implementation with the static creation method that returns an <code class="ph codeph">AuthInitialize</code> object. The method obtains credentials for clients to a cluster. The obtained credentials should be acceptable to the <code class="ph codeph">SecurityManager</code> specified through the security-manager property on the peers.</td>
```
Comment (jdeppe-pivotal) [inline geode-docs/reference/topics/gemfire_properties.html.md.erb:571]: How about: `A reference to a static creation method that...` and then... `The method returns credentials for clients to authenticate and connect to a cluster.`

**Review by davebarnes97 — APPROVED** [review]
Doc change looks good.
I'll defer to others on the technical aspects.

**Comment by karensmolermiller** [comment]
Reviewers, please re-review.  This JIRA has been expanded to correct the docs surrounding authentication. I also found another section that had partial, but incomplete duplicate information. I have revised that other section to remove the repeated (but incomplete/incorrect) info, and instead have links to where the info is. My goal is that the docs have a single place where the info resides; a single location is far easier to maintain.

**Review by jdeppe-pivotal — APPROVED** [review]
Looks good. I didn't realize you can also just give a class name to the `*-auth-init` properties.

**Review by davebarnes97 — APPROVED** [review]
+1  A major upgrade to the explanation of this tricky topic. Nice work!

**Pull request merged**